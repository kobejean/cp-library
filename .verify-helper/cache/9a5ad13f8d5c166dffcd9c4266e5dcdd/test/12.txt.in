50
1 564
0 513
2 939
0 182
2 674
0 0
3 839
2 347
2 601
3 876
0 257
0 597
2 845
2 70
2 641
3 464
2 809
2 141
0 940
0 36
0 695
0 9
2 771
2 428
2 404
2 571
0 399
3 28
2 190
0 863
0 72
1 266
0 304
2 378
1 223
1 356
3 453
2 699
1 615
0 235
2 380
2 182
0 949
1 941
1 301
3 214
1 930
0 886
3 477
3 85
