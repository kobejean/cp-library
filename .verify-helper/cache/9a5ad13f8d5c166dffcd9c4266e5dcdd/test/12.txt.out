474.499999999999999972
